### ISSUE TITLE
⛔ 🧩 🐞 Bug Report: Using custom template in post-widget (all Elementor) missing alt text in frontend are missing

### ISSUE THREAD

**Issue by Ringsoe** [elementor/elementor#21577]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi Elementor, 
You guys are doing a great job and I have been using your Elementor for several years. However, I have noticed a persistent issue in the program/plugin. The text I enter in the alt field does not appear on mouseover in the frontend! This happens every time the content I display is loaded through a template. Please check out my website, where you can see it in action. The tooltip shows the website title and a number instead of the alt text. I hope this is understandable. If not, please feel free to contact me. 
Best regards 🙂
Website: www.boerglumparken.dk or www.ringsoe.dk

### Steps to reproduce

1) Insert posts widget
2) Choose custom layout
3) Create template
4) Insert image widget with alt text
5) Save

### Isolating the problem

- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 44, Revision 27f408641db v5.7.41-44
	PHP Version: 7.4.33
	PHP Memory Limit: 2048M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://www.boerglumparken.dk
	Home URL: https://www.boerglumparken.dk
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 2048M
	Max Memory limit: 2048M
	Permalink Structure: /%postname%/
	Language: da-DK
	Timezone: 1
	Admin Email: torben@ringsoe.dk
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: da_DK
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Bing Webmaster Url Submission
		Version: 1.0.13
		Author: Bing Webmaster

	Duplicate Page
		Version: 4.5.1
		Author: mndpsingh287

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Elementor
		Version: 3.11.2
		Author: Elementor.com

	Elementor Addon Elements
		Version: 1.12.1
		Author: WPVibes

	Elementor Pro
		Version: 3.11.3
		Author: Elementor.com

	Loco Translate
		Version: 2.6.3
		Author: Tim Whitlock

	NitroPack
		Version: 1.5.19
		Author: NitroPack LLC

	Premium Addons for Elementor
		Version: 4.9.51
		Author: Leap13

	Rank Math SEO
		Version: 1.0.109
		Author: Rank Math

	Search Exclude
		Version: 1.3.1
		Author: QuadLayers

	Simple Calendar
		Version: 3.1.37.2
		Author: Simple Calendar

	SiteGround Security
		Version: 1.4.1
		Author: SiteGround

	Smash Balloon Instagram Feed
		Version: 6.1.2
		Author: Smash Balloon

	Sticky Header Effects for Elementor
		Version: 1.5.5
		Author: Rwattner

	Templately
		Version: 2.1.0
		Author: Templately

	Visibility Logic for Elementor
		Version: 2.3.3
		Author: StaxWP

== Elements Usage ==
	
	custom_grid : 1
		ele-loop-item : 5
	footer : 2
		heading : 4
		icon-list : 4
		image : 1
		social-icons : 1
		text-editor : 1
	loop : 10
		ele-loop-item : 1
		image : 1
		post-info : 1
		theme-post-excerpt : 1
		theme-post-title : 1
	section : 17
		heading : 1
	widget : 0
		gallery : 1
		premium-addon-blog : 4
		premium-countdown-timer : 1
	wp-post : 222
		animated-headline : 1
		blockquote : 1
		button : 55
		divider : 1
		eae-info-circle : 1
		facebook-embed : 2
		gallery : 1
		heading : 636
		icon : 2
		icon-box : 9
		image : 268
		image-box : 3
		lottie : 1
		media-carousel : 1
		post-info : 54
		posts : 1
		premium-addon-blog : 62
		premium-addon-button : 1
		premium-addon-dual-header : 1
		premium-addon-image-separator : 1
		premium-addon-video-box : 1
		premium-countdown-timer : 54
		premium-icon-list : 1
		shortcode : 62
		slides : 1
		spacer : 1
		table-of-contents : 54
		text-editor : 1078
		theme-post-excerpt : 1
		video : 1
	wp-page : 5
		button : 2
		call-to-action : 4
		divider : 2
		heading : 10
		icon-box : 2
		image : 11
		image-box : 2
		post-info : 3
		posts : 15
		premium-addon-blog : 1
		premium-addon-button : 1
		premium-addon-dual-header : 3
		search-form : 1
		shortcode : 1
		slides : 1
		template : 1
		text-editor : 6
		theme-post-title : 3
	single-post : 4
		divider : 3
		heading : 1
		post-info : 1
		posts : 9
		spacer : 1
		theme-post-content : 1
	loop-item : 1
		icon-box : 1

== Indstillinger ==
	
	cpt_support: post, page, calendar
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1

== Funktioner ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrationer ==
	

== Elementor Experiments ==
	Optimeret DOM-output: Aktiv
	Forbedret ressourceindlæsning : Inaktiv som standard
	Forbedret indlæsning af CSS ressourcer: Inaktiv som standard
	Inline Font Icons: Inaktiv
	Forbedringer af tilgængeligheden: Inaktiv
	Additional Custom Breakpoints: Inaktiv som standard
	admin_menu_rearrangement: Inaktiv som standard
	Flexbox Container: Inaktiv som standard
	Upgrade Swiper Library: Inaktiv som standard
	Default to New Theme Builder: Aktiv som standard
	Hello sidehoved og sidefod: Aktiv
	Editor V2: Inaktiv som standard
	Landingsside: Aktiv
	Nested Elements: Inaktiv som standard
	Lazy Load baggrundsbilleder: Inaktiv som standard
	Gem som standard: Aktiv som standard
	Page Transitions: Aktiv som standard
	Notes: Aktiv som standard
	Loop: Aktiv som standard
	Modtagne formularer: Aktiv
	Scroll Snap: Aktiv som standard
	Mega Menu: Inaktiv som standard

== Log ==
	
Log: showing 20 of 362023-02-16 13:23:25 [info] elementor::elementor_updater Started 
2023-02-16 13:23:25 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-16 13:23:25 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 13:23:26 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 13:23:27 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-16 13:23:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-23 11:10:02 [info] elementor::elementor_updater Started 
2023-02-23 11:10:02 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-23 11:10:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-23 11:10:04 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-23 11:10:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-23 18:37:30 [info] elementor-pro::elementor_pro_updater Started 
2023-02-23 18:37:30 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-23 18:37:30 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-23 18:37:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-27 11:26:14 [info] elementor-pro::elementor_pro_updater Started 
2023-02-27 11:26:14 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-27 11:26:14 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-27 11:26:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.2',
  'to' => '3.11.3',
)]
2023-02-27 11:26:14 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.2',
  'to' => '3.11.3',
)]

JS: showing 3 of 3JS: 2023-02-01 10:13:41 [error X 4][https://www.boerglumparken.dk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:1075365] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-02-13 08:43:06 [error X 1][https://www.boerglumparken.dk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:739072] Cannot read properties of null (reading 'config') 
JS: 2023-03-04 20:03:08 [error X 5][https://www.boerglumparken.dk/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,underscore,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:2:51824] Cannot read properties of undefined (reading 'replace') 

PHP: showing 3 of 3PHP: 2023-02-13 14:01:22 [notice X 2][/var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-15 13:50:20 [notice X 4][/var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-02-19 17:03:56 [notice X 4][/var/www/ringsoe.dk/boerglumparken.dk/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Ele Custom Skin: Inkompatibel
	Elementor Addon Elements: Inkompatibel
	Elementor Pro: Kompatibel
	Premium Addons for Elementor: Kompatibel
	Sticky Header Effects for Elementor: Kompatibilitet ikke specificeret
	Visibility Logic for Elementor: Inkompatibel

== Elementor Pro - Compatibility Tag ==
	
	Ele Custom Skin: Inkompatibel
	Elementor Addon Elements: Inkompatibel
	Premium Addons for Elementor: Kompatibel
	Visibility Logic for Elementor: Inkompatibel

**Comment by nicholaszein** [comment]
Hello @Ringsoe,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- I noticed you are using Rank Math for SEO. It is likely what's altering your alt tags.

### ✅ What you can do to solve your issue:
- Check the Rnk Math settings or documentation.

### 👍 For now, that's all we can help with. 

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
